file_and_modules.md

```diff
@@ -1,10 +1,9 @@
-## File operations
-* file is a storage format where we store information on the computer.
-* It might text files like music, video, code and other stuff or binary
-  files which are only understood able by computer.
+## Filesystem operations
+* A file is stored as a resource on the computer on top of a [filesystem](https://www.tldp.org/LDP/sag/html/filesystems.html) which is a storage format.
+* Files can be text , music or media or any binaries which can only be understood by computer.
```

Comment:
There are two types of files, one which is understandable by user and by computer by like this. Add like this.

Reply:
done!